src/lib.rs

```diff
@@ -22,9 +22,9 @@ pub use askar_storage as storage;
 pub use askar_storage::future;
 
 #[cfg(feature = "ffi")]
-mod ffi;
+pub mod ffi;
 
 pub mod kms;
 
-mod store;
+pub mod store;
```

Comment:
What's in the store module that's not being exposed?

Reply:
This is the symbols I use from store.
```pub use aries_askar::store::{entry, PassKey, Session, Store, StoreKeyMethod};```

Reply:
Can't you just `use aries_askar::{entry, PassKey, Session, Store, StoreKeyMethod}` though?

Reply:
Yeah. It works.